Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -53,14 +53,19 @@ public static class BasicOrchardFeaturesTestingUITestContextExtensions
     /// specified <paramref name="setupRecipeId"/>.
     /// </para>
     /// <para>The test method assumes that the site is not set up.</para>
+    /// <para>When running headless version of Orchard Core, ContentOperations shall be excluded.
+    /// Utilize <paramref name="dontCheckFrontend"></paramref>> for this purpose.</para>
```

Comment:
Add the "the" and format the other occasions similarly.

```suggestion
    /// <para>
    /// When running the headless version of Orchard Core, ContentOperations shall be excluded. Utilize <paramref
    /// name="dontCheckFrontend"></paramref>> for this purpose.
    /// </para>
```

Reply:
This isn't addressed.

Reply:
Do you mean change this to `Utilize **the** <paramref name="dontCheckFrontend">` ?

Reply:
Got it.